src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const elements = document.querySelectorAll('.population');
+const totalResult = document.querySelector('.total-population');
+const averageResult = document.querySelector('.average-population');
+
+const stringsArr = [...elements].map(el => el.textContent);
+const numArr = stringsArr.map(el => +el.split(',').join(''));
+
+const total = numArr.reduce(
+  (accumulator, current) => accumulator + current, 0,
+);
```

Comment:
```suggestion
const total = numArr.reduce((accumulator, current) => accumulator + current, 0);
```
do not start callback arguments from the new line